twtr/src/integration-test/groovy/twtr/AccountFunctionalTests/BadAccountValuesSpec.groovy

```diff
@@ -0,0 +1,156 @@
+package twtr.AccountFunctionalTests
+
+import geb.spock.GebSpec
+import grails.test.mixin.integration.Integration
+import groovyx.net.http.HttpResponseException
+import groovyx.net.http.RESTClient
+import org.apache.http.HttpResponse
+import spock.lang.Shared
+import spock.lang.Stepwise
+import spock.lang.Unroll
+
+@Integration
+@Unroll
+class BadAccountValuesSpec extends GebSpec {
+    final static String goodHandle = 'scsu_huskies'
+    final static String goodEmail = 'testemail@test.com'
+    final static String goodPassword = 'abc123ABC'
+    final static String goodDisplayName = 'SCSU Huskies'
+
+    final static String goodHandle2 = 'scsu_huskies2'
+    final static String goodEmail2 = 'testemail2@test.com'
+
+    RESTClient restClient
+
+    @Shared
+    def goodId
+
+    @Shared
+    def startingAccountCount
+
+    def setup() {
+        restClient = new RESTClient('http://localhost:8080')
+
+        goodId = restClient.post(path: '/accounts', contentType: 'application/json',
+                body: [handle  : goodHandle, emailAddress: goodEmail,
+                       password: goodPassword, displayName: goodDisplayName]).data?.id
+
+        startingAccountCount = restClient.get(path: '/accounts').data?.size()
```

Comment:
Nice. I assume that there are no existing accounts for all of my functional tests.